test/EmptyApp.FunctionalTests/project.json

```diff
@@ -1,31 +1,30 @@
 ﻿{
   "webroot": "wwwroot",
-  "version": "1.0.0-*",
+  "version": "1.0.0-rc2",
```

Comment:
All the tests except MVC  test applications are passing now. Seems like, I am missing some configuration change/update with respect to MVC test application.